marshmallow_utils/schemas/identifier.py

```diff
@@ -85,7 +85,10 @@ def validate_identifier(self, data, **kwargs):
 
         # Validate scheme
         if not scheme:
-            errors["scheme"] = self.error_messages["required"]
+            if identifier:
+                errors["scheme"] = self.error_messages["invalid_scheme"]
+            else:
```

Comment:
from the code I understand that if there is no scheme but there is an identifier, we still display "invalid scheme" ? This does not sound right, unless I am missing context. My understanding is that only if you have both scheme and identifier, you can validate the identifier, otherwise you don't know against which scheme you should be validating the id